engine/spec/components/targeted_content_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+RSpec.shared_examples "public_targeted_content" do
```

Comment:
traditionally you would place these inside rspec support. Usually your folder structure would be

```
spec
  support
    shared_contexts
      stuff
    shared_examples
      stuff
```

One file for each shared example. Usually named accordingly. So this one could be `public_targeted_content`

Reply:
Good to know. I reckon I'll keep them inlined for now as there's only one set and used within this file. Buy we should totally do that if we end up with more of these that cross specs 👍